src/compiler/scala/tools/nsc/transform/CleanUp.scala

```diff
@@ -524,4 +584,24 @@ abstract class CleanUp extends Statics with Transform with ast.TreeDSL {
 
   } // CleanUpTransformer
 
+
+  val BOX = termNames.box
```

Comment:
Within the compiler, we use `nme` instead of the longer alias `termNames`.

Reply:
done